packages/auth/src/signature/middleware.ts

```diff
@@ -146,21 +153,15 @@ export async function tokenHttpsigMiddleware(
     ctx.params['id']
   )
   if (!accessToken) {
-    throwGNAPError(
-      ctx,
+    throw new GNAPServerRouteError(
       401,
       GNAPErrorCode.InvalidClient,
       'invalid access token'
     )
   }
 
   if (!accessToken.grant) {
-    const logger = await ctx.container.use('logger')
-    logger.error(
-      `access token with management id ${ctx.params['id']} has no grant associated with it.`
-    )
```

Comment:
is this log worth keeping still?

Reply:
Probably...in hindsight that's a pretty severe case that shouldn't happen - we should know about if it does for some reason